apps/mobile/src/screens/register-adoption/index.tsx

```diff
@@ -0,0 +1,153 @@
+          placeholder={t("REGISTER_ADOPTION.FORM.OBSERVATIONS_PLACEHOLDER")}
+          multiline
+          numberOfLines={2}
+          mode="outlined"
+          value={text}
+          onChangeText={(text) => setText(text)}
+        />
+        <Subheading>
+          {t("REGISTER_ADOPTION.FORM.TYPE_OPTIONS.LABEL")}
+        </Subheading>
+        <View style={styles.typesContainer}>
+          {["DOG", "CAT", "OTHER"].map((type) => (
+            <Button
+              key={type}
+              mode="outlined"
+              onPress={() => console.log("first")}
+              style={{
+                borderRadius: 100,
+                borderColor: theme.colors.primary,
+                borderWidth: 2,
+              }}
+            >
+              {t(`REGISTER_ADOPTION.FORM.TYPE_OPTIONS.${type}`)}
+            </Button>
+          ))}
+        </View>
+        <Subheading>{t("REGISTER_ADOPTION.FORM.AGE")}</Subheading>
+        <RNSlider />
+        <Subheading style={{ color: theme.colors.primary }}>
+          1 {t("YEAR")}
+        </Subheading>
+        <RadioButton.Group
+          onValueChange={(value) => setChecked(value)}
+          value={checked}
+        >
+          <Subheading>{t("REGISTER_ADOPTION.FORM.SEX.LABEL")}</Subheading>
+          <RadioButton.Item
+            label={t("REGISTER_ADOPTION.FORM.SEX.MALE")}
+            value="first"
+          />
+          <RadioButton.Item
+            label={t("REGISTER_ADOPTION.FORM.SEX.FEMALE")}
+            value="second"
+          />
+        </RadioButton.Group>
+        <Subheading>{t("REGISTER_ADOPTION.FORM.SIZE.LABEL")}</Subheading>
+        <View style={styles.typesContainer}>
+          {["SMALL", "MEDIUM", "BIG"].map((type) => (
+            <Button
+              key={type}
+              mode="outlined"
+              onPress={() => console.log("first")}
+              style={{
+                borderRadius: 100,
+                borderColor: theme.colors.primary,
+                borderWidth: 2,
+              }}
+            >
+              {t(`REGISTER_ADOPTION.FORM.SIZE.${type}`)}
+            </Button>
+          ))}
+        </View>
+        <Button
+          mode="contained"
+          color={theme.colors.accent}
+          style={{ elevation: 0 }}
+          labelStyle={{ padding: 10, color: theme.colors.onPrimary }}
+          onPress={() => console.log("first")}
+        >
+          {t("REGISTER_ADOPTION.FORM.CONFIRM_BUTTON")}
+        </Button>
+      </form>
+    </Container>
+  );
+}
+
+const styles = StyleSheet.create({
+  titleContainer: {
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+    justifyContent: "space-between",
+    marginBottom: 28,
+  },
+  formContainer: {
+    display: "flex",
+    position: "relative",
+    flexDirection: "column",
+    gap: 24,
+  },
+  typesContainer: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-around",
+  },
+});
```

Comment:
Not sure if the task was clear to you, but we wanna split this whole form into a multistep one. Also, let's use a form state manager and I highly recommend [react-hook-form](https://react-hook-form.com/). We've started an [article](https://blog.codeminer42.com/the-art-of-multistep-forms-1-navigation/) with an approach we came up with to structure multistep forms using this library.